opencog/attention/MinMaxSTIUpdatingAgent.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * opencog/attention/MinMaxSTIUpdatingAgent.h
+ *
+ * Copyright (C) 2008 by OpenCog Foundation
+ * Written by Joel Pitt <joel@fruitionnz.com>
```

Comment:
Honestly i wasn't quite ready to push these changes to master. This is already on my list of todos. But misgana wanted me to make a pullrequest. But i am just very busy with univiersity right now.